### PR TITLE
Layout adjustments 

### PR DESCRIPTION
**Pull request by tissyane**
- The userPosts Page now has the same layout as Timeline Page (including Trending box and NavBar);
- Exported components from Timeline Page to reuse on userPage;
- Adjusted the height of post boxes and added overflow to it;
- Using the same default image (on image error) for Publish Post Box, userPosts Page and NavBar;
- Changed message color on "Loading" and "Error" at Timeline and User Pages;
- Removed scroll on Timeline Page and setted height to 100% to prevent white block during "Loading" or "Error";
- Removed unnecessary console.log;
- Adjusted the top margin of the Trending box.
- Fixed the title size on mobile.

### CODE REVIEW

File: src/components/pages/UserTimeline.js [hunk 1]
```diff
@@ -31,28 +30,41 @@ function UserTimeline() {
...
-        ))
-      ) : (
-        <p>{message}</p>
-      )}
-      <Trending />
+      <Navbar />
+      <Wrapper>
+        <Feed>
+          <Title>
+            {/* <Image picture={userPosts.picture} /> */}
```
Comment (henriquesdias) [inline src/components/pages/UserTimeline.js]: Need to remove the comment

**Review by henriquesdias — APPROVED** [review]
Nice work

**Comment by MacenaAndre** [comment]
Great!!

File: src/components/pages/UserTimeline.js (hunk 1 above)
Comment (tissyane) [reply src/components/pages/UserTimeline.js]: Ooooh nooo! I really forgot this one. Thank you. Already removed it

**Review by MacenaAndre — APPROVED** [review]

**Pull request merged**